.github/workflows/release-please.yml

```diff
@@ -0,0 +1,31 @@
+name: release-please
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This will release a new version on every push to main. Do we really want that, or should we go for a different approach?

Reply:
This runs the action on every push to main but I don't think it will release every time. On every push to main it creates/updates a PR for the next release (like [this one](https://github.com/nodejs/node-addon-api/pull/1526)) and the release is only published once this PR is merged.